internal/db/rdb/mysql/dbr/dbr.go

```diff
@@ -0,0 +1,56 @@
+//
```

Comment:
can you add package doc?

Reply:
@kevindiu @hlts2 
updated :)
https://github.com/vdaas/vald/pull/659/commits/d3b35ca4c6c9e35c84596db19f7a91a1c76273c2